src/core/Match.cpp

```diff
@@ -17,6 +17,12 @@
 
 #include <time.h>
 
+#ifdef _MSC_VER
+// suppress unknown attribute warning
+#pragma warning(disable : 5051)
```

Comment:
Hm. On VS2015, this is warning 5030. Where did 5051 come from?
Also, this pragma should go into something more low-level, we don't want to reintroduce it whenever we use [[fallthrough]]...

Reply:
I've got warning C5051 on VS2019 Community Edition. Rechecked with Clang, and is also warn about usage of this attributes with `std=c++11` (but understand them), so I've commented these attributes and removed these pragmas. gcc understand "special comments" (https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wimplicit-fallthrough) and `\\[[fallthrough]];` processed in `-Wimplicit-fallthrough=2` mode (clang don't use comments and will warn if `-Wimplicit-fallthrough` enabled). It's also possible to use special attributes for clang and gcc in c++11 mode, but I'm not sure, if we want to duplicate them for each compiler (another way is to add macro that will use correct attribute for each compiler).